lightningd/peer_htlcs.c

```diff
@@ -621,11 +621,45 @@ static void forward_htlc(struct htlc_in *hin,
 {
 	const u8 *failmsg;
 	struct lightningd *ld = hin->key.channel->peer->ld;
-	struct channel *next = active_channel_by_scid(ld, scid);
+	struct channel *next;
 	struct htlc_out *hout = NULL;
 
+	/* This is a shortcut for specifying next peer; doesn't mean
+	 * the actual channel! */
+	next = any_channel_by_scid(ld, scid);
+	if (next) {
+		struct peer *peer = next->peer;
+		struct channel *channel;
+		struct amount_msat best_spendable = channel_amount_spendable(next);
+
+		/* Seek largest capacity channel! */
```

Comment:
A bit of a nomenclature mixup I think: capacity to me always sounds like the total funds in the channel, whereas balance is the current allocation to either, though we don't have any rules around nomenclature.

Reply:
Yes!  I wrote the comment then changed my mind when I wrote the code :)